CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Highlights are marked with a pancake 🥞
 ### Changed
 
 - One global state object and operation graph for all groups [#781](https://github.com/p2panda/p2panda/pull/781)
+- Improvements to strong remove resolver [#788](https://github.com/p2panda/p2panda/pull/788)
```

Comment:
The CHANGELOG entry (and PR title) can be made more meaningful. You already have a good template in your PR description:

> Handle several new edge cases in strong remove resolver, improve concurrent re-add logic and handle cycles in nested groups.

How about: "Handle strong removal edge cases and cycles in nested groups, improve concurrent re-adds"